tests/test_python_version.py

```diff
@@ -0,0 +1,34 @@
+"""
+Ensure all the various places in which we need to reference our target Python version
+are consistent.
+"""
+import re
+from pathlib import Path
+
+import pytest
+
+
+BASE_DIR = Path(__file__).parents[1]
+
+PYTHON_VERSION = "3.11"
```

Comment:
what if we made `.python-version` the canonical version, and then we reuse this script across a few different repos?

Reply:
(I mean actually reuse, i.e. put the script in a shared place)

Reply:
Ah yeah, making `.python-version` canonical is a good idea. I'm not sure we've got a good mechanism for sharing this kind of thing yet though. And I suspect the details of which files need to be checked and how will end up being quite project specific. I'm definitely in favour of spreading the pattern around though.

Reply:
Have updated to use `.python-version` as the source-of-truth.

Reply:
thanks